src/infrastructure/jira/jira-client/jira-client.ts

```diff
@@ -72,8 +77,12 @@ class JiraClient {
 	getIssue = async (
 		issueIdOrKey: string,
 		connectInstallation: ConnectInstallation,
-	): Promise<GetIssueResponse> =>
-		withAxiosErrorTranslation(async () => {
+	): Promise<GetIssueResponse> => {
+		const context = {
+			baseUrl: connectInstallation.baseUrl,
+			clientKey: connectInstallation.clientKey,
```

Comment:
During investigating a different issue we realised that it would really help if we also logged `issueIdOrKey`. Since you're already on it, could you please add `issueIdOrKey` to the context?

Reply:
Yes - I added issueIdOrKey. 

I also added propertyKey as well in case it's helpful.